cmd/dump/dump.go

```diff
@@ -16,9 +16,10 @@ func main() {
 	var duration time.Duration
 	pflag.CommandLine.AddGoFlagSet(kubecli.FlagSet())
 	pflag.DurationVarP(&duration, "since", "s", 10*time.Minute, "collection window, defaults to 10 minutes")
+	maxFails := pflag.Int("maxfails", 10, "max failed tests to report, defaults to 10")
```

Comment:
Ups, just realized that this flag does not make any sense at all. The `dump` binary just reused the per-test dump code, but its purpose is to just dump once and then exit. That can be removed and a hardcoded `1` should be good enough.

Reply:
Yes I wasn't sure what to put there. 
Will change it and hardcode it to 1 with an explanatory comment.

Reply:
Done, I also did a bit of cleanup so the invocations from dump are more explicit.